setup.py

```diff
@@ -13,20 +15,26 @@ def read(fname):
     return open(os.path.join(os.path.dirname(__file__), fname)).read()
 
 
-class NPMInstall(build_py):
+class MyBuildCommand(build_py):
     def run(self):
         call(["npm", "install", "--prefix", "fhir2dataset/metadata"])
         build_py.run(self)
 
 
+class MySdistCommand(sdist):
+    def run(self):
+        call(["npm", "install", "--prefix", "fhir2dataset/metadata"])
+        sdist.run(self)
+
+
 requirements = read("requirements.txt").split()
 
 setup(
-    cmdclass={"build_py": NPMInstall},
+    cmdclass={"build_py": MyBuildCommand, "sdist": MySdistCommand},
```

Comment:
Do we need both?

Reply:
I've asked myself that a lot. And clearly, I feel that I lack knowledge about packaging (I couldn't find a tutorial that suited my lack of knowledge). As a consequence, I'm not clear about the differences between: 
* `python setup.py install`
* `python setup.py build`
* `python setup.py sdist`
I've read several things diagonally, including this section: https://pythonhosted.org/MRV/develop/distribute/setup.html#intimacies-of-sdist-build-py-and-install. Basically what I understand is that it's better if all 3 commands can work.
As `build` and `install` call build_py, I figured I could change it, and since `sdist` is separate, I made a separate command.

Reply:
Fine by me! I do agree that the 3 can be useful 🙂

Reply:
Thanks !!